lib/scrape.js

```diff
@@ -165,6 +166,45 @@ async function findOrganization({
   return null
 }
 
+async function findWiki(name) {
+  const APIurl = wdk.searchEntities({
+    search: name,
+    format: 'json',
+  })
+  const srch = await fetch(APIurl)
+  const response = await srch.json()
+  if (response && response.search.length) {
+    const wikidataurl = response.search[0].url
+    const wikidataid = response.search[0].id
+    const entityAPIurl = wdk.getEntities({
+      ids: [wikidataid],
+      props: ['sitelinks'],
+      format: 'json',
+    })
+
+    const sitelinks = await fetch(entityAPIurl)
+    let sitelink = await sitelinks.json()
+    sitelink = sitelink['entities'][wikidataid]['sitelinks']
+    if (sitelink && sitelink['enwiki']) {
```

Comment:
I think here you should definitely do `sitelink.enwiki`

Reply:
In future(with `i18n`) this should become language_code + 'wiki'. So that'd probably bring us back to this way...

Reply:
Ah, gotcha, that makes sense.